google/cloud/security/scanner/scanner.py

```diff
@@ -227,18 +230,27 @@ def _output_results(all_violations, **kwargs):
     """
 
     # Write violations to database.
+    (inserted_rows, violation_errors) = (0, [])
     try:
         vdao = violation_dao.ViolationDao()
-        vdao.import_violations(all_violations, kwargs.get('snapshot_timestamp'))
+        (inserted_rows, violation_errors) = vdao.insert_violations(
+            all_violations, snapshot_timestamp=snapshot_timestamp)
     except db_errors.MySQLError as err:
         LOGGER.error('Error importing violations to database: %s', err)
 
-    # Write the CSV.
+    # TODO: figure out what to do with the errors. For now, just log it.
+    LOGGER.debug('Inserted %s rows with %s errors',
+                 inserted_rows, len(violation_errors))
```

Comment:
super small nit: "inserted_rows" is an number, rather than actual "row"; so it would read better if this is "inserted_row_count" or "insertion_count")